### PR TITLE
Add Timers table and connect som_timer db

### PR DESCRIPTION
**Pull request by sienna-kopf**
Co-Authored-By: Dorion <60492885+sciencefixion@users.noreply.github.com>
Co-Authored-By: Chandler Hulstrom <62038636+Chulstro@users.noreply.github.com>

# SOM-Timer BE Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- connect Postgres database to the Flask App 
- [relevant tutorial](https://stackabuse.com/using-sqlalchemy-with-flask-and-postgresql/)

Create Database:
```
$ psql 
your_name=# CREATE DATABASE som_timer;
```

[Install Python](https://www.python.org/downloads/mac-osx/) Latest version of 3 </br>
[Install Flask](https://flask.palletsprojects.com/en/1.1.x/installation/#install-create-env):
```
$ cd som_timer_be
$ python3 -m venv venv
$ . venv/bin/activate
```

Install Dependencies (while in venv):
```
$ pip install psycopg2-binary
$ pip install flask-sqlalchemy
$ pip install Flask-Migrate
```

Run Migrations:
```
$ export FLASK_APP=api.py
$ flask db init
$ flask db migrate
$ flask db upgrade
```

Look @ your fancy new table in Postico (or your preferred database management application)!! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`

## Has This Been Tested?
- [x] No
- [ ] Yes

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs/pry sessions
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: migrations/alembic.ini [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+[handler_console]
+class = StreamHandler
+args = (sys.stderr,)
+level = NOTSET
+formatter = generic
+
+[formatter_generic]
+format = %(levelname)-5.5s [%(name)s] %(message)s
+datefmt = %H:%M:%S
```
Comment (Chulstro) [inline migrations/alembic.ini:45]: Excited to see what all this means!

**Review by Chulstro — APPROVED** [review]

File: migrations/alembic.ini (hunk 1 above)
Comment (Abdeboskey) [reply migrations/alembic.ini:45]: That makes at least 2 of us!

File: migrations/env.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
+
```
Comment (Abdeboskey) [inline migrations/env.py:2]: Y'all are importing code from the future?

File: migrations/alembic.ini (hunk 1 above)
Comment (rwilliams659) [reply migrations/alembic.ini:45]: Same here!

**Review by rwilliams659 — APPROVED** [review]

**Review by Abdeboskey — APPROVED** [review]
I'm really impressed that y'all got this spun up so quickly. I'm definitely struggling to follow a lot of what's happening here, would love a little tour when we can find the time.

**Review by sciencefixion — APPROVED** [review]

**Comment by sciencefixion** [comment]
The README is beautiful! 🤩

**Review by jkwest-93 — APPROVED** [review]

**Comment by jkwest-93** [comment]
Nice work y'all! Glad this is coming together well. Great documentation as well!

**Pull request merged**